libs/wire-api/src/Wire/API/Error/Brig.hs

```diff
@@ -142,6 +143,13 @@ type instance MapError 'CodeAuthenticationFailed = 'StaticError 403 "code-authen
 
 type instance MapError 'CodeAuthenticationRequired = 'StaticError 403 "code-authentication-required" "Code authentication is required"
 
+type instance
+  MapError 'MissingLegalholdConsentOldClients =
+    'StaticError
+      403
+      "missing-legalhold-consent-old-clients"
+      "Failed to connect to a user or to invite a user to a group because somebody is under legalhold and somebody else has old clients that do not support the UI requirements"
```

Comment:
```suggestion
      "Failed to connect to a user or to invite a user to a group because somebody is under legalhold and somebody else has old clients that do not support legalhold's UI requirements"
```